frontend/components/customizer/customizer.js

```diff
@@ -18,16 +18,12 @@ export default class Customizer extends React.Component {
     exampleNamedParams: objectOfKeyValuesPropType,
     exampleQueryParams: objectOfKeyValuesPropType,
     initialStyle: PropTypes.string,
+    showActualParams: PropTypes.bool,
+    link: PropTypes.string,
```

Comment:
Would it be better to pass this in through `exampleQueryParams`?

Reply:
Good idea. The `link` is moved to query params in all places. This have one consequence - `link` is automatically added to a badge url and to a customization form:

![image](https://user-images.githubusercontent.com/1526000/56825805-8f2dce80-685a-11e9-94ef-201d6b7bcffb.png)
![image](https://user-images.githubusercontent.com/1526000/56825823-9b199080-685a-11e9-91ea-42218218fdac.png)

Of course we can change it somehow, but I would like to hear your opinion.

Reply:
This is just in the suggested badges, right? I think that is just fine! 😁